.goreleaser.yml

```diff
@@ -21,6 +21,17 @@ build:
   goarm:
     - 7
   ldflags: -s -w -X github.com/astronomer/astro-cli/version.CurrVersion={{ .Version }} -X github.com/astronomer/astro-cli/version.CurrCommit={{ .Commit }}
+brews:
+  - github:
+      owner: astronomer
+      name: homebrew-tap
+    folder: Formula
+    homepage:  https://astronomer.io
+    description: To build and run Airflow DAGs locally via docker-compose and deploy DAGs to Astronomer-managed Airflow clusters and interact with the Astronomer API.
+    test: |
+      system "#{bin}/astro version"
+    dependencies:
+    - name: go
```

Comment:
let me test... i don't think we need golang as deps, cause your brew formula take binaries:
https://github.com/astronomer/homebrew-tap/blob/master/Formula/astro.rb#L8-L10